UKHomeOffice/nmsw-ui
# NMSW-423 Update link text on request new email page

# Ticket

NMSW-423

----

## AC

Change the wording for the details link

----

## To test

- Run app
- Go to localhost:3000
- Click on 'create an account'
- Enter an email and submit
- Click on 'Not received an email?'
- The content should match what is in the prototype
----

## Notes
